### ISSUE TITLE
Request: Include libspeex codec

### ISSUE THREAD

**Issue by titoudoc** [oznu/ffmpeg-for-homebridge#75]
**Describe The Bug:**
Nestcam plugin error in homebrige : missing libspeex codec

**To Reproduce:**
Install ffmpeg-for-homebridge make Nest-cam pluggin work (prior error was saying FFMEG is missing).
Camera stream is visible  but I can't send audio to the camera via iPhone microphone. 
Error show in RED in log : This version of FFMPEG does not support the audio codec 'libspeex'. You may need to recompile FFMPEG using '--enable-libspeex' and restart homebridge.

**Expected behavior:**
Working bidirectionnal audio

**ffmpeg output:**

```
ffmpeg -version
ffmpeg version 6.0-homebridge-alpine-armv7l-static Copyright (c) 2000-2023 the FFmpeg developers
built with gcc 12.2.1 (Alpine 12.2.1_git20220924-r10) 20220924
configuration: --enable-hardcoded-tables --enable-nonfree --enable-gpl --enable-hardcoded-tables --enable-hardcoded-tables --enable-openssl --enable-libdav1d --enable-libsvtav1 --enable-libx264 --enable-libxvid --enable-libvidstab --enable-libzimg --enable-lv2 --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libmp3lame --enable-libopus --enable-libvorbis --enable-libtheora --enable-libfdk-aac --enable-libwebp --enable-libsrt --disable-debug --disable-shared --enable-pthreads --enable-static --enable-version3 --extra-cflags='-I/build/workspace/include -I/build/workspace/include/lilv-0' --extra-ldexeflags=-static --extra-ldflags=-L/build/workspace/lib --extra-libs='-ldl -lpthread -lm -lz' --pkgconfigdir=/build/workspace/lib/pkgconfig --pkg-config-flags=--static --prefix=/build/workspace --extra-version=homebridge-alpine-armv7l-static
libavutil      58.  2.100 / 58.  2.100
libavcodec     60.  3.100 / 60.  3.100
libavformat    60.  3.100 / 60.  3.100
libavdevice    60.  1.100 / 60.  1.100
libavfilter     9.  3.100 /  9.  3.100
libswscale      7.  1.100 /  7.  1.100
libswresample   4. 10.100 /  4. 10.100
libpostproc    57.  1.100 / 57.  1.100
```

**Screenshots:**
![Sans titre 2](https://github.com/homebridge/ffmpeg-for-homebridge/assets/46103481/35979c9c-0285-474f-8708-8b4b2ec73afd)

**Environment:**

* **Operating System**: cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 12 (bookworm)"
NAME="Raspbian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=raspbian
ID_LIKE=debian
* **Homebridge version**: 1.7.0
* **Node.js version**: v20.11.0

**Comment by hjdhjd** [comment]
This isn’t a bug but an enhancement request. Will look into it for future releases. Thanks for the request.

**Comment by titoudoc** [comment]
Ok thanks, should I submit an enhancement request ?

**Comment by hjdhjd** [comment]
Already took care of it. Thanks!

**Comment by n0rt0nthec4t** [comment]
Any chance to update the build after all these months with libspeex? Not a major change to include the library in the build

**Comment by hjdhjd** [comment]
PRs are always welcome. The builds will be updated when time allows for it...it's not a trivial process, timewise.

TL;DR: always a chance. They'll be updated as time allows.

**Comment by n0rt0nthec4t** [comment]
Ok.. just testing one now, including bumping to ffmpeg 6.1.2

**Comment by hjdhjd** [comment]
Enjoy. It’s harder than it looks. 😄

**Comment by n0rt0nthec4t** [comment]
https://github.com/homebridge/ffmpeg-for-homebridge/pull/80

**Comment by n0rt0nthec4t** [comment]
Completed under 04cee63338a962723862a83c29d2501a5d5fcccb

**Comment by hjdhjd** [comment]
Included in v2.1.2. Thanks for the contributions.

**Comment by n0rt0nthec4t** [comment]
No worries. Thanks for the fast version release update
